Source/WebCore/dom/StyledElement.cpp

```diff
@@ -192,6 +193,11 @@ void StyledElement::invalidateStyleAttribute()
 
     Node::invalidateStyle(validity);
 
+    if (isSVGElement()) {
+        if (auto* svgElement = dynamicDowncast<SVGElement>(this))
+            svgElement->invalidateInstances();
+    }
```

Comment:
Should not `StyledElement ::invalidateStyleAttribute()` be made virtual? In this case this code can be moved to `SVGElement::invalidateStyleAttribute()` after calling the base `StyledElement ::invalidateStyleAttribute()`?